impl/src/test/java/com/sun/faces/mock/MockBeanManager.java

```diff
@@ -263,4 +263,14 @@ public Collection<Context> getContexts(Class<? extends Annotation> scopeType) {
         return null;
     }
 
+    @Override
```

Comment:
Update the copyright header to include 2024?

Reply:
Thanks, I think I did that, but the file didn't save or so. Oh well, it's done now ;)